driver.js

```diff
@@ -9,21 +9,15 @@ const Wappalyzer = require("./wappalyzer");
 const { setTechnologies, setCategories, analyze, analyzeManyToMany, resolve } =
   Wappalyzer;
 
-const { CHROMIUM_BIN, CHROMIUM_DATA_DIR, CHROMIUM_WEBSOCKET, CHROMIUM_ARGS } =
-  process.env;
+const { CHROMIUM_BIN, CHROMIUM_WEBSOCKET, CHROMIUM_ARGS } = process.env;
 
 const chromiumArgs = CHROMIUM_ARGS
   ? CHROMIUM_ARGS.split(" ")
   : [
-      "--headless",
-      "--single-process",
       "--no-sandbox",
-      "--no-zygote",
       "--disable-gpu",
-      "--ignore-certificate-errors",
-      "--allow-running-insecure-content",
-      "--disable-web-security",
-      `--user-data-dir=${CHROMIUM_DATA_DIR || "/tmp/chromium"}`,
+      "--disable-dev-shm-usage",
```

Comment:
With these changes in general, I'd just want us to make sure this doesn't impact the results of what we find, which I don't think it should. But would be worth testing with a few big scans to refresh and look for changes in the components found. Related to this are we able to switch back to the previous version of this library if we need to?